src/ray/protobuf/gcs.proto

```diff
@@ -409,6 +409,8 @@ message JobsAPIInfo {
   // The driver process exit code after the driver executed. Return None if driver
   // doesn't finish executing
   optional int32 driver_exit_code = 14;
+  // The amount of reservable memory resource in bytes for the entrypoint command.
+  optional double entrypoint_memory = 15;
```

Comment:
It's a bit unexpected that this is a double, because it refers to a number of bytes.  Can you double check this?

Reply:
(We should match whatever is used by `ray.init()` and the `memory` argument to `@ray.remote()`)